clustergroup/templates/_helpers.tpl

```diff
@@ -202,13 +202,19 @@ kind: OperatorGroup
 metadata:
   name: {{ $k }}-operator-group
   namespace: {{ $k }}
+      {{- if (hasKey $v "targetNamespaces") }}
+      {{- if $v.targetNamespaces }}
```

Comment:
Isn't this functionally the same as `hasKey $v "targetNamespaces"`?

Do you need both?

Reply:
@strangiato The default behavior is that if you do not have `targetNamespaces` defined, but you have `operatorGroup: true`, then we will add the namespace to the OperatorGroup manifest like so:

```yaml
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: openshift-distributed-tracing-operator-group
  namespace: openshift-distributed-tracing
spec:
  targetNamespaces:
    - openshift-distributed-tracing
 ```
So in order for us to make it a global OperatorGroup you have to define both the `operatorGroup: true` and the `targetNamespaces: []` or `targetNamespaces: null`.  Hope that makes sense.

Reply:
Without the first if, that checks for null, the template explodes.
Error: error calling len: len of nil pointer